scripts/asm-installer/install_asm

```diff
@@ -214,6 +225,11 @@ FLAGS:
   -e|--enable_apis                    Allow this script to enable necessary APIs
                                       on your behalf. Without this flag, it will
                                       abort if APIs are not already enabled.
+     --print_config                   Instead of installing ASM, print all of
+                                      the compiled YAML to stdout. All other
+                                      output will be written to stderr, even if
+                                      it would normally go to stdout. Skip all
```

Comment:
Not sure I agree about "skip all validations" in this case. Is that standard/idiomatic for this kind of command? We can discuss offline.

Reply:
I agree that it's a trade-off. In the context of the linked issue (GitOps / CI/CD integration) I thought it was safer for now to disable it by default. I don't know if the CI/CD user/service account is going to have the access to check everything, if the cluster names may be templated in later, etc. @therealmitchconnors do you have any guidance one way or the other?

Reply:
Is this validation against user-supplied config, or against runtime values in the cluster or project?

Reply:
Runtime values. The CLI options/dependencies are still verified.

Reply:
Then it does seem like a runtime validation might not be possible in some CI scenarios.  Perhaps we should have a separate flag to control this?

Reply:
Sure. Added #176 to track the extra flag.